spec/factories/plate_factories.rb

```diff
@@ -98,13 +145,19 @@
       end
     end
 
+    # Sets up a plate of GBS requests with configured primer panels
+    # Use pool_sizes to determine how many wells have requests
     factory :v2_plate_with_primer_panels do
+      initialize_with do
+        Sequencescape::Api::V2::Plate.load(attributes)
+      end
       transient do
         purpose_name { 'Primer Panel example' }
         request_factory { :gbs_library_request }
       end
     end
 
+    # Sets up a plate of 2 isd requests, complete with bait libraries
```

Comment:
What's an isd request?

Reply:
Typo, should have said ISC (Indexed Sequence Capture). Fixed typo and added expanded version of the abbreviation to comment.